src/Components/GrowthChart/GrowthChartBuilder/GrowthChartBuilder.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import i18n from '@dhis2/d2-i18n';
 import { Line } from 'react-chartjs-2';
+import { ChartOptions } from 'chart.js';
 import 'chart.js/auto';
 import { ChartDataTypes } from '../../../types/chartDataTypes';
 import { chartLineColorPicker } from '../../../utils/chartLineColorPicker';
 import { annotateLineEnd } from '../../../utils/annotateLineEnd';
 
 export const GrowthChartBuilder = ({
-    dataSetValues, dataSetMetadata, xLabelValues, keysDataSet,
+    dataSetValues, dataSetMetadata, xAxisValues, yAxisValues, keysDataSet,
```

Comment:
Have each of the props on a new line
```suggestion
    dataSetValues,
    dataSetMetadata,
    xAxisValues,
    yAxisValues,
    keysDataSet,
```